pages/api/route.js

```diff
@@ -0,0 +1,56 @@
+import { NextResponse } from "next/server";
+

```

Comment:
No need for API_KEY working with Backend serv er usually used to integrate with third-party systems.